### PR TITLE
Onboard API version 2017-08-01

### PR DESCRIPTION
**Pull request by shihying**
## Description

Onboard API version 0801, regenerate ASAzure SDK based on a 0801 version swagger spec.

PR:
https://github.com/Azure/azure-rest-api-specs/pull/2306
Merged Commit:
https://github.com/Azure/azure-rest-api-specs/commit/cb9c18b988dd67894653a34dc55b7978403b120a
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/AnalysisServices/Management.Analysis/Microsoft.Azure.Management.Analysis.csproj [hunk 1]
```diff
@@ -6,9 +6,9 @@
 	<PropertyGroup>
 		<Description>Provides management capabilities for Azure Analysis Services accounts.</Description>
 		<AssemblyTitle>Microsoft Azure Analysis Services Management Library</AssemblyTitle>
-		<Version>2.0.2</Version>
+		<Version>2.0.3</Version>
 		<AssemblyName>Microsoft.Azure.Management.Analysis</AssemblyName>
-		<PackageId>ManagementAnalysis;Analysis</PackageId>
+		<PackageId>Microsoft.Azure.Management.Analysis</PackageId>
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/Management.Analysis/Microsoft.Azure.Management.Analysis.csproj]: Add package release notes

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
do the following in below order
1) Generate SDK using generate.cmd (this add required meta data file)
2) Once SDK generated, build your project as below
msbuild build.proj /t:build /p:Scope=SDKs\AnalysisServices
this will add second meta data file that is required.
Update the PR

File: src/SDKs/AnalysisServices/Management.Analysis/Microsoft.Azure.Management.Analysis.csproj (hunk 1 above)
Comment (shihying) [reply src/SDKs/AnalysisServices/Management.Analysis/Microsoft.Azure.Management.Analysis.csproj]: thanks!

**Comment by shihying** [comment]
@shahabhijeet

Thanks for reminder, have used generate.cmd and rebuild to generate metadata file
Also updated release note.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**